android/app/src/main/kotlin/com/jabook/app/jabook/audio/data/repository/PlaylistRepository.kt

```
@@ -63,6 +64,7 @@ public class PlaylistRepository
                 playlistDao.upsertPlaylist(entity)
                 Result.Success(Unit)
             } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)